server/startup/cdn.js

```diff
@@ -0,0 +1,12 @@
+import { Meteor } from "meteor/meteor";
+/**
+ * Sets prefix for the user who wants to serve bundled js and css
+ * files from different URL
+ *
+ * @returns null
+ */
+export default function CDN() {
```

Comment:
Function naming should follow the conventions in the style guide. Specifically this should be a verb giving you some sense of what this function does and camel casing includes acronyms. e.g. `setupCdn` . And it should include a JSDoc

Reply:
@Akarshit What's happening with this?